app/mailers/enrollment_mailer.rb

```diff
@@ -24,6 +28,15 @@ class EnrollmentMailer < ActionMailer::Base
           "contact@api.gouv.fr"
         end
 
+      @provider = case enrollment.fournisseur_de_donnees
+        when "franceconnect" then "FranceConnect"
```

Comment:
we should not repeat this enumeration for the 3rd time. We could factorize this in one hash instead. The result will be: 

```rb
sender = mailParams[enrollment.fournisseur_de_donnees].sender
provider = mailParams[enrollment.fournisseur_de_donnees].provider
```